packages/express-did-auth/test/authentication-factory.test.ts

```diff
@@ -51,71 +54,100 @@ describe('authenticationFactory', () => {
     await testAuthFactory(state)(req, res)
   })
 
-  test('should respond with 401 if the subject of the challenge response is not the service did', async () => {
+  test('should respond with 401 if the subject of the challenge response is not the service url', async () => {
     MockDate.set(modulo0Timestamp)
 
     const challenge = challengeVerifier.get(userIdentity.did)
-    const anotherIdentity = await identityFactory()
 
-    const challengeResponseJwt = await challengeResponseFactory(challenge, userIdentity, anotherIdentity.did, config.serviceUrl)
+    const challengeResponseJwt = challengeResponseFactory(challenge, userIdentity, userPrivateKey, 'https://taringa.net')
 
     const req = { body: { response: challengeResponseJwt } }
-    const res = mockedResFactory(401, CORRUPTED_CHALLENGE)
+    const res = mockedResFactory(401, INVALID_CHALLENGE_RESPONSE)
 
     const logic = mockBusinessLogicFactory(false)
     await testAuthFactory(state, logic)(req, res)
   })
 
-  test('should respond with 401 if extra business logic that returns false', async () => {
+  test('should respond with 401 if the subject of the signer is not the given did', async () => {
     MockDate.set(modulo0Timestamp)
 
     const challenge = challengeVerifier.get(userIdentity.did)
-    const challengeResponseJwt = await challengeResponseFactory(challenge, userIdentity, config.serviceDid, config.serviceUrl)
+    const anotherIdentity = await identityFactory()
+
+    const challengeResponseJwt = challengeResponseFactory(challenge, anotherIdentity.identity, anotherIdentity.privateKey, 'https://taringa.net')
```

Comment:
```suggestion
    const challengeResponseJwt = challengeResponseFactory(challenge, anotherIdentity.identity, anotherIdentity.privateKey, config.serviceUrl)
```
Let's use the proper serviceUrl so we can test that is validating just the signer

Reply:
Uh sorry. Copy paste 🤦